actions/responses.go

```diff
@@ -95,26 +93,9 @@ func ResponsesCreate(c buffalo.Context) error {
 		return errors.WithStack(err)
 	}
 
-	for _, a := range response.Answers {
```

Comment:
This is not needed because the answers are automatically being saved by `tx.ValidateAndCreate(response)` above?

Reply:
seems to be the case, yes